drivers/modem/Kconfig.ublox-sara-r4

```diff
@@ -60,6 +65,19 @@ config MODEM_UBLOX_SARA_R4_APN
 	  for the network connection context.  This value is specific to
 	  the network provider and may need to be changed.
 
+config MODEM_UBLOX_SARA_AUTODETECT_APN
+	bool "detect APN automatically"
+	help
+	  Enable automatic detection of the APN, based on the IMSI
+	  If the detection fails, the configured APN will be used
+
+config MODEM_UBLOX_SARA_AUTODETECT_APN_PROFILES
+	string "list of profiles to search when autodetecting APN"
+	default "hologram=23450, wm=20601 29505 29509 23450 53703 90143"
+	help
+		Set a comma separated list of profiles, each containing of:
```

Comment:
Please indent the help lines like the other help blocks, so with two spaces

Reply:
done